src/AnyOfCodeGenerator/AnyOfCodeGenerator.cs

```diff
@@ -397,6 +397,8 @@ private static StringBuilder AddHeader()
         sb.AppendLine("// </auto-generated>");
         sb.AppendLine("//------------------------------------------------------------------------------");
         sb.AppendLine();
+        sb.AppendLine("#pragma warning disable CS1591");

```

Comment:
Also add `#pragma warning restore CS1591` at the end of the code to restore the warnings.
Like as defined here:
[CS1591](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/suppress-warnings#use-a-preprocessor-directive)

Reply:
`#pragma warning restore` is only needed if the instruction is meant to be scoped to specific lines of code within a file. That is not the case here - the warning is intended to be disabled for each entire file.

> The disable takes effect beginning on the next line of the source file. The warning is restored on the line following the restore. If there's no restore in the file, the warnings are restored to their default state at the first line of any later files in the same compilation.

(See [#pragma warning](https://learn.microsoft.com/dotnet/csharp/language-reference/preprocessor-directives#pragma-warning))

Reply:
Thanks